filebeat/input/docker/input.go

```diff
@@ -68,8 +69,8 @@ func NewInput(
 		return nil, errors.New("Docker input requires at least one entry under 'containers.ids'")
 	}
 
-	for idx, containerID := range ids {
-		cfg.SetString("paths", idx, path.Join(config.Containers.Path, containerID, "*.log"))
+	for idx, containerID := range config.Containers.IDs {
```

Comment:
Is there a reason to don't use `ids`? It filters empty docker ids.

Reply:
Well, here, again, I just used the code that was here when I created my repo. I'll see if using ids is fine.
Thanks.